src/app/modules/convai-checker/convai-checker.component.ts

```diff
@@ -82,6 +81,11 @@ export interface DemoSettings {
   // The model to use for scoring. If not specified, uses TOXICITY as the
   // default model.
   modelName?: string;
+
+  // The font style to use in the plugin. This should match what is used in a
+  // 'font-family' CSS rule. Note that we have only tested the following fonts:
```

Comment:
Out of curiosity, what happens if this doesn't match what it's the `font-family` CSS rule? Just wondering if that's something that we need to make more explicit for whoever may override this.

Reply:
The font would revert to whatever the default browser font is. It's not great, but I can't think of another way to have a custom font, unless we wanted to hardcode an enum of options

Reply:
Does that mean anyone hoping to use the "fontFamily" option also has to change the CSS? If that's the case I'm not sure I understand the value of adding the option (as opposed to something like a [SASS mixin](https://sass-lang.com/documentation/at-rules/mixin)).

Reply:
No they don't have to do anything to the CSS other than make sure the font they choose is loaded on their webpage. They just have to specify this property and we inject it into the ngStyle for the component (otherwise they couldn't do it without using /deep/ selectors which is frowned upon.) We're going to offer a dropdown of pre-tested font options in the UI which will populate this field without the user needing to know CSS and we can pre-load those. More advanced users can customize this field on their own.

Reply:
Ah, that makes sense. Thanks for the explanation!